docusaurus/docs/snippets/plugin-e2e-ds-workflow.pnpm.md

```diff
@@ -0,0 +1,92 @@
+```yaml title=".github/workflows/e2e.yml"
+name: E2E tests
+on:
+  pull_request:
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  resolve-versions:
+    name: Resolve Grafana images
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    outputs:
+      matrix: ${{ steps.resolve-versions.outputs.matrix }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Resolve Grafana E2E versions
+        id: resolve-versions
+        uses: grafana/plugin-actions/e2e-version@main
+
+  playwright-tests:
+    needs: resolve-versions
+    timeout-minutes: 60
+    strategy:
+      fail-fast: false
+      matrix:
+        GRAFANA_IMAGE: ${{fromJson(needs.resolve-versions.outputs.matrix)}}
+    name: e2e ${{ matrix.GRAFANA_IMAGE.name }}@${{ matrix.GRAFANA_IMAGE.VERSION }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: pnpm/action-setup@v2
+        with:
+          version: ^6.10.0
```

Comment:
This version is quite old and was never supported by create-plugin. If we only care about supporting our tooling I think we can drop it as the [action supports reading the version](https://github.com/pnpm/action-setup?tab=readme-ov-file#version) from the package.json packageManager property which we set on scaffold and update.

Reply:
Yep sounds good, will fix.